040-SDK/111-search.mdx

```diff
@@ -7,48 +7,55 @@ slug: sdk/search
 published: true
 ---
 
-As you insert data into Xata, it is automatically indexed for full-text search. You can run a search by using the [/search endpoint](/docs/api-reference/db/db_branch_name/tables/table_name/search#free-text-search-in-a-table). While the `/query` endpoint exists only at the table level, the `/search` endpoint exists both at the database branch and the table level, because it is possible to search across table.
+Xata natively provides text-based search for your database with the following core features:
 
-The search index is updated asynchronously after each insert/update, meaning that the search results are eventually consistent with the results that you get from the `/query` endpoint. Another fundamental difference to the `/query` endpoint is that `/search` doesn't support following links. This means that for links records you can filter by the ID of the linked record, but not any of the other columns. If you need filter or search by linked columns, other than the ID, it is recommended that you denormalize the data.
+- Fuzzy matching to account for typos.
```

Comment:
Would suggest parallelism for list 

- Fuzzy matching for typo tolerance in search queries
- Query limits to specified tables and columns
- Pre-search filters for searching within specific data subsets
- Weights and boosts to influence the relevance of search scores based on certain columns
- Highlight markup in search results to emphasize partial matches in returned content

Reply:
> Query limits to specified tables and columns

A limit can mean something in programming, so this read as "Query limit". The parameter is also named `filter`, which is why that language was used. Reworded it to the below to match your sentence structure.

> Targets allow isolation of specific tables and columns in search queries